src/lib/variants.ts

```diff
@@ -67,6 +67,16 @@ export interface Test {
     // they are always 0 and 1?
     audience?: number;
     audienceOffset?: number;
+
+    // These are specific to hardcoded tests
+    start?: string;
```

Comment:
I don't _think_ I've ever seen this in frontend, but should there also be an equivalent check to `isNotExpired` that checks whether we're before the start of the test?

Reply:
I don't think it's in use either, the assumption is that the test becomes active when you deploy the changes (and flick the switch if you're using one)

Reply:
i.e. unlike `expiry`, `start` is meta only.